function.tf

```diff
@@ -83,16 +155,20 @@ resource "google_cloudfunctions_function_iam_member" "cloud_scheduler" {
 }
 
 resource "google_cloud_scheduler_job" "this" {
-  count = var.enable_function ? 1 : 0
-
-  name        = var.name
+  name        = "${var.name}-job"
   description = "Triggers the Cloud Function"
-  schedule    = var.function_schedule
+  schedule    = var.function_schedule_frequency
 
   http_target {
     http_method = "POST"
     uri         = google_cloudfunctions_function.this[0].https_trigger_url
 
+    headers = {
+      Content-Type = "application/json"
+    }
+
+    body        = base64encode("{}")
```

Comment:
Having to send an empty JSON object is unexpected, why is this needed?

Reply:
At one point I was sending GCP resource types, and asset types as arguments because I thought customer might want to disable / enable. Turns out there's no costs / downsides to doing everything but the result is (currently) the Python code is expecting a body that is json